physionet-django/templates/about/database_content.html

```diff
@@ -23,785 +23,41 @@ <h2>Overview</h2>
 <p>
 This page displays a curated list of databases in the PhysioNet archives. To find more databases on PhysioNet, <a href="{% url 'content_index' %}">search our resources</a>.</p>
 <ul>
-<li> <a href="#clinical">Clinical Databases</a>: Data from critical
-       care clinical settings that may include demographics, vital
-       sign measurements made at the bedside, laboratory test results,
-       procedures, medications, caregiver notes, images and imaging
-       reports, and mortality (both in and out of hospital).</li>
-<li><a href="#waveform">Waveform Databases</a>: High resolution continuous recordings of physiological signals. Waveform databases are organized according to their signal and annotation types:
-<ul>
-  <li> <a href="#multi">Multi-Parameter Databases</a>:
-     Available signals vary, but may include ECG, continuous invasive blood
-     pressure, respiration, oxygen saturation, and EEG, among others.</li>
-  <li> <a href="#ecg">ECG Databases</a>.  Also see
-     <a href="#multi">Multi-Parameter Databases</a>, most of which include
-     ECG signals.</li>
-  <li> <a href="#rr">Interbeat (RR) Interval Databases</a>:
-     These contain beat annotations obtained from ECG recordings, but the
-     ECG signals are not available.  Also see <a href="#ecg">ECG Databases</a>,
-     most of which include beat annotations in addition to the original ECG
-     signals.</li>
-  <li> <a href="#cv">Other Cardiovascular Databases</a></li>
-  <li> <a href="#gait">Gait and Balance Databases</a></li>
-  <li> <a href="#neuro">Neuroelectric and Myoelectric Databases</a>: EEG, EHG, and more.</li>
-  <li> <a href="#synthetic">Synthetic Data</a></li>
-</ul>
-<li> <a href="#image">Image Databases</a></li>
-<li> <a href="#miscellaneous">Miscellaneous Databases</a></li>
+<li> <a href="#open">Open Access</a>: Data projects with open access.</li>
+<li> <a href="#restricted">Restricted Access</a>: Data projects with restricted access.</li>
+<li> <a href="#credentialed">Credentialed Access</a>: Data projects with credentialed access.</li>
 </li>
 </ul>
 
 <p>
-Each database is placed into a class according to the following specifications. Contributed data are placed in classes 2 and 3 on acceptance, and may be admitted to class 1 after review and a public comment period.</p>
-<ul>
-<li>Class 1 - completed reference databases.</li>
-<li>Class 2 - archival copies of raw data that support published research, contributed by authors or journals.</li>
-<li>Class 3 - other contributed collections of data, including works in progress.</li>
-</ul>
-
-<p>
-On this page, listings within each group are ordered by class, and then
+On this page, listings within each group are ordered by its access policy, and then
```

Comment:
This following sentence is difficult to follow:

> "On this page, listings within each group are ordered by its access policy, and then..."

Perhaps delete the sentence, and begin the overview with:

> This page displays a complete list of databases, grouped by access policy. To search for content, visit [the search page](https://physionet.org/content/).